### ISSUE TITLE
Cannot download rgb_images.zip from google drive

### ISSUE THREAD

**Issue by XinchaoGou** [valeoai/WoodScape#33]
Hi guys

Thanks for providing the dataset! 
I had a problem when I tried to download the rgb_images.zip from google drive.
![image](https://user-images.githubusercontent.com/35408641/118911813-5ee18d00-b959-11eb-8ede-eebaf67ac999.png)
I tried a few times and every time it failed with this forbidden error.
![image](https://user-images.githubusercontent.com/35408641/118912195-fb0b9400-b959-11eb-83a6-ef4e1761acb6.png)

Best regards,
Xinchao

**Comment by G453** [comment]
Hi Xinchao,
Have you checked if its due to the restrictions from your service provider or organization ?
I verified the download link and its working fine ( from a laptop & an android mobile as well). Could you please check from other PC or by connecting to another network ?  

Thanks & Regards,
Ganesh

**Comment by XinchaoGou** [comment]
Hi Ganesh

Thank you for your prompt reply!
Yes I used this URL [https://drive.google.com/drive/folders/1X5JOMEfVlaXfdNy24P8VA-jMs0yzf_HR](https://drive.google.com/drive/folders/1X5JOMEfVlaXfdNy24P8VA-jMs0yzf_HR)
My colleagues and I tried few times on different computers. It worked fine at the beginning but failed after a while. 
We thought the file was too big, so we tried to download subdirectories.
I successfully downloaded some of these subdirectories as shown in the picture which are crossed out.
![image](https://user-images.githubusercontent.com/35408641/118965886-bacd0580-b99b-11eb-96de-f75ef131fc25.png)

I even downloaded over 11GB of rgb_images.zip, but it still failed in the end.
I'll take your advice and try it again. If I make any progress, I'll notify you.

Best Regards,
Xinchao
